src/main/java/core/basesyntax/service/FruitTransactionParser.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface FruitTransactionParser {
+    public List<FruitTransaction> transformToTransaction(List<String> operations);
```

Comment:
You already have an enum Operation. Is this it? If not, let's come up with a better name for the parameter